node/public/script/main.js

```diff
@@ -33,14 +33,14 @@ if (window.location.pathname == "/") {
 
     //on attend le signal init du serveur qui envoie la main et le board
     socket.on('init', function (init) {
-
+        console.log(init.board);
         //on parcours le board pour l'afficher
         for (indexBoard = 0; indexBoard < 4; indexBoard++) {
             for (indexBoard2 = 0; indexBoard2 < 6; indexBoard2++) {
                 let tmp = indexBoard + 1;
                 let tmp2 = indexBoard2 + 1;
                 if (init.board[indexBoard][indexBoard2] != undefined) {
-                    $('.l' + tmp + 'col' + tmp2).html("<img src='src/" + init.board[indexBoard][indexBoard2].value + ".png' alt='test'></img>");
+                    $('.l' + tmp + 'col' + tmp2).html("<img src='src/" + init.board[indexBoard][indexBoard2].value + ".png' alt='testfzefz'></img>");
```

Comment:
Rather than having a meaningless alt attribute, think about accessibility and visually impaired people using screen readers and remove it. https://davidwalsh.name/accessibility-tip-empty-alt-attributes